python/dgl/nn/tensorflow/hetero.py

```diff
@@ -88,6 +88,11 @@ class HeteroGraphConv(layers.Layer):
     >>> print(y_dst.keys())
     dict_keys(['user', 'game'])
 
+    .. note::
```

Comment:
Have you tried building the doc and see if it gets rendered properly with `.. note::`?